src/config/kernel.md

```diff
@@ -105,71 +83,54 @@ file needs to be created like `/etc/modprobe.d/radeon.conf` with the contents:
+run manually:
 
 ```
-$ xbps-query -Rs dkms
-[-] acpi_call-dkms-1.2.0_2             Kernel module allowing calls to ACPI methods through /proc/acpi/call
-[-] dkms-2.7.1_1                       Dynamic Kernel Modules System
-[-] exfat-dkms-1.2.8_5                 Exfat kernel driver (nofuse)
-[-] lttng-modules-dkms-2.10.9_2        LTTng modules provide Linux kernel tracing capability
-[-] openrazer-driver-dkms-2.5.0_1      Kernel driver for Razer devices (DKMS-variant)
-[-] rtl8812au-dkms-20190731_1          Realtek 8812AU/8821AU USB WiFi driver (DKMS)
-[-] rtl8822bu-dkms-20190427_1          Realtek 8822BU USB WiFi driver (DKMS)
-[-] spl-0.7.13_1                       Solaris Porting Layer -- userland and kernel modules (using DKMS)
-[-] tp_smapi-dkms-0.43_1               IBM ThinkPad hardware functions driver
-[-] vhba-module-dkms-20190410_1        Virtual (SCSI) HBA module used by cdemu
-[-] virtualbox-ose-dkms-6.0.10_1       General-purpose full virtualizer for x86 hardware - kernel module sources for dkms
-[-] virtualbox-ose-guest-dkms-6.0.10_1 General-purpose full virtualizer for x86 hardware - guest addition module source for dkms
-[-] zfs-0.8.1_1                        Z File System -- userland, pyzfs, and kernel modules (using DKMS)
-[-] zfs-32bit-0.8.1_1                  Z File System -- userland, pyzfs, and kernel modules (using DKMS) (32bit)
-[-] broadcom-wl-dkms-6.30.223.271_8    Broadcom proprietary wireless drivers for Linux - DKMS kernel module
-[-] catalyst-dkms-15.302_2             AMD catalyst driver 15.12 for Linux - DKMS kernel module
-[-] nvidia-dkms-430.14_2               NVIDIA drivers for linux - DKMS kernel module
-[-] nvidia340-dkms-340.107_3           NVIDIA drivers (GeForce 8, 9, 9M, 100, 100M, 200, 300 series) - DKMS kernel module
-[-] nvidia390-dkms-390.116_3           NVIDIA drivers (GeForce 400, 500 series) - DKMS kernel module
+# xbps-reconfigure --force linuxX.X
 ```
 
+If run manually, they serve to apply initramfs configuration changes to the next
+boot.
+
+### Remove hooks
+
+The `{pre,post}-remove` hooks are executed by
+[vkpurge(8)](https://man.voidlinux.org/vkpurge.8) when it's used to remove old
+kernels.
+
+## Dynamic Kernel Module Support (dkms)
+
+There are kernel modules that are not part of the Linux source tree that are
+built at install time using dkms and [kernel hooks](#kernel-hooks). The
+available modules can be listed by searching for `dkms` in the package
+repositories.
+
 ## cmdline
 
 ### GRUB
 
-Kernel command line arguments can be added through the grub bootloader by
+Kernel command line arguments can be added through the GRUB bootloader by
 editing `/etc/default/grub` and changing the `GRUB_CMDLINE_LINUX_DEFAULT`
-variable and then regenerating the grub configuration.
-
-```
-# vi /etc/default/grub
-# grub-mkconfig -o /boot/grub/grub.cfg
-```
+variable and then reconfiguring the `grub` package you have installed.
```

Comment:
Suggest removing "you have installed".

Reply:
I don't use GRUB, so I didn't know where the kernel hooks came from. Seems it's from the `grub` package, not the specific ones, like `grub-x86_64-efi`.

Fixed!